test/qpy_compat/test_qpy.py

```diff
@@ -242,6 +243,18 @@ def generate_single_clbit_condition_teleportation():  # pylint: disable=invalid-
     return teleport_qc
 
 
+def generate_parameter_vector():
+    """Generate tests for parameter vector element ordering."""
+    qc = QuantumCircuit(11)
+    input_params = ParameterVector("x_par", 11)
+    user_params = ParameterVector("θ_par", 11)
+    for i, param in enumerate(user_params):
+        qc.ry(param, i)
+    for i, param in enumerate(input_params):
+        qc.rz(param, i)
+    return qc
+
+
```

Comment:
Wouldn't hurt to add the "vector element in parameter expression" test as a test case in this section as well, I think.

Reply:
Done in: https://github.com/Qiskit/qiskit-terra/pull/7381/commits/c93e1a7f1652729f4b1b3e256467f54c3cef699f